### PR TITLE
[main] Adding `prometheus-node-exporter` and `prometheus-process-exporter`.

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

Adding `prometheus-node-exporter` and `prometheus-process-exporter` to `SPECS`.

###### Change Log  

- Added `prometheus-node-exporter` and `prometheus-process-exporter`.

###### Does this affect the toolchain?  

No.

###### Test Methodology

- Local package builds.
- Installed packages inside a local CBL-Mariner container.

### CODE REVIEW

File: SPECS/prometheus-node-exporter/prometheus-node-exporter.spec [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+install -Dpm0644 %{SOURCE2} %{buildroot}%{_sysusersdir}/%{name}.conf
+install -Dpm0644 %{SOURCE3} %{buildroot}%{_unitdir}/%{name}.service
+install -Dpm0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/default/%{name}
+install -Dpm0644 example-rules.yml %{buildroot}%{_datadir}/prometheus/node-exporter/example-rules.yml
+install -Dpm0644 %{SOURCE5} %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
+mkdir -vp %{buildroot}%{_sharedstatedir}/prometheus/node-exporter
+
+%check
+make test
```
Comment (mfrw) [inline SPECS/prometheus-node-exporter/prometheus-node-exporter.spec:77]: Wondering if the `make test` respects the `vendor` folder or tries to fetch stuff from the internet ?

File: SPECS/prometheus-process-exporter/prometheus-process-exporter.spec [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ln -s %{name} %{buildroot}%{_bindir}/process-exporter
+
+install -Dpm0644 %{SOURCE2} %{buildroot}%{_unitdir}/%{name}.service
+install -Dpm0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
+install -Dpm0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/default/%{name}
+
+mkdir -vp %{buildroot}%{_sharedstatedir}/prometheus
+
+%check
+make test integ
```
Comment (mfrw) [inline SPECS/prometheus-process-exporter/prometheus-process-exporter.spec:75]: Does the `make test` respect the `vendor` directory and use vendored mods or downloads them from the internet ?
[same question as above]

**Review by mfrw — APPROVED** [review]
LGTM.
A couple of clarification requested though :)

**Review by jslobodzian — APPROVED** [review]

File: SPECS/prometheus-node-exporter/prometheus-node-exporter.spec (hunk 1 above)
Comment (PawelWMS) [reply SPECS/prometheus-node-exporter/prometheus-node-exporter.spec:77]: I've sent you a message with some details offline.:)

File: SPECS/prometheus-process-exporter/prometheus-process-exporter.spec (hunk 2 above)
Comment (PawelWMS) [reply SPECS/prometheus-process-exporter/prometheus-process-exporter.spec:75]: Same response as above.;)

**Pull request merged**